### PR TITLE
[idrac_redfish_storage_controller] - Enhancement for 'Secure Erase' operation

### PR DESCRIPTION
**Pull request by ABHISHEK-SINHA10**
# Description
A few sentences describing the overall goals of the pull request's commits.

# GitHub Issues
List the GitHub issues impacted by this PR:

| GitHub Issue # |
| -------------- |
| |

# ISSUE TYPE
- Feature Pull Request

##### COMPONENT NAME
idrac_redfish_storage_controller

##### OUTPUT

```paste below

```
##### ADDITIONAL INFORMATION

```paste below

```
# Checklist:

- [x] I have performed a self-review of my own code to ensure there are no formatting, vetting, linting, or security issues
- [x] I have verified that new and existing unit tests pass locally with my changes
- [x] I have not allowed coverage numbers to degenerate
- [x] I have maintained at least 90% code coverage
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have maintained backward compatibility

### CODE REVIEW

**Comment by ansible-collections-svc** [comment]
Can one of the admins verify this patch?

**Review by sachin-apa — COMMENTED** [review]
Update Version in module and test file ay the copywrites.

**Review by sachin-apa — DISMISSED** [review]

**Review by ShivamSh3 — DISMISSED** [review]

File: plugins/modules/idrac_redfish_storage_controller.py [hunk 1]
```diff
@@ -184,7 +196,7 @@
   - "Jagadeesh N V (@jagadeeshnv)"
   - "Felix Stephen (@felixs88)"
   - "Husniya Hameed (@husniya_hameed)"
-  - "Abhishek Sinha (@Abhishek-Dell)"
+  - "Abhishek Sinha (@ABHISHEK-SINHA10)"
 notes:
     - Run this module from a system that has direct access to Dell iDRAC.
     - This module is supported on iDRAC9.
```
Comment (rajshekarp87) [inline plugins/modules/idrac_redfish_storage_controller.py:202]:
```suggestion
    - This module is supported on iDRAC9.
    - This module supports IPv4 and IPv6 addresses.
```

File: plugins/modules/idrac_redfish_storage_controller.py [hunk 2]
```diff
@@ -169,13 +176,18 @@
...
+       and when I(command) is C(SecureErase).
     type: bool
     default: false
   job_wait_timeout:
     description:
       - The maximum wait time of job completion in seconds before the job tracking is stopped.
       - This option is applicable when I(job_wait) is C(true).
+      - "Note: When I(command) is C(SecureErase), If a secure erase needs a reboot,
+        the job will get scheduled and waits for no of seconds specfied in I(job_wait_time),
+        to reduce the wait time either give I(job_wait_time) minimum or make I(job_wait)
```
Comment (rajshekarp87) [inline plugins/modules/idrac_redfish_storage_controller.py]:
```suggestion
        to reduce the wait time either give I(job_wait_time) a lesser value or make I(job_wait)
```

File: roles/idrac_storage_controller/meta/argument_specs.yml [hunk 3]
```diff
@@ -106,6 +106,12 @@ argument_specs:
               - Assigns a global hot spare or unassigns a hot spare.
               - C(true) assigns the disk as a global hot spare.
               - C(false) unassigns the disk as a hot spare.
+          erase:
+            type: bool
+            description:
+              - Securely erase a device.
```
Comment (rajshekarp87) [inline roles/idrac_storage_controller/meta/argument_specs.yml]: Add the same documentation as in module

File: roles/idrac_storage_controller/meta/argument_specs.yml [hunk 4]
```diff
@@ -106,6 +106,12 @@ argument_specs:
...
+              - C(true) securely erase the disk.
```
Comment (rajshekarp87) [inline roles/idrac_storage_controller/meta/argument_specs.yml:113]: Add an example for this under "Example Playbook"

**Review by sachin-apa — DISMISSED** [review]

File: roles/idrac_storage_controller/meta/argument_specs.yml (hunk 3 above)
Comment (ABHISHEK-SINHA10) [reply roles/idrac_storage_controller/meta/argument_specs.yml]: Done

**Review by ShivamSh3 — DISMISSED** [review]

**Review by rajshekarp87 — APPROVED** [review]

**Review by ShivamSh3 — APPROVED** [review]

**Review by sachin-apa — APPROVED** [review]

**Pull request merged**